content/blog/2019-01-31-more-editors.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: more_editors
+title: 'Announcing new software review editors: Melina Vidoni and Brooke Anderson'
+date: '2019-01-31'
+authors:
+  - Stefanie Butland
+categories: blog
+topicid:
+tags:
+  - Software Peer Review
+  - rOpenSci team
+---
+
+We are pleased to welcome Brooke Anderson and Melina Vidoni to our team of [Associate Editors](https://ropensci.github.io/dev_guide/softwarereviewintro.html#associateditors) for rOpenSci [Software Peer Review](https://ropensci.org/software-review/). They join Scott Chamberlain, Anna Krystalli, Lincoln Mullen, Karthik Ram, Noam Ross and Maëlle Salmon. With the addition of Brooke and Melina, our editorial board now includes four women and four men, located in North America, South America and Europe.
+
+Our open Software Peer Review system for community-contributed R tools is a key component of [our mission](/about/) to create technical infrastructure that lowers barriers to working with data sources on the web. Editors manage the review process, performing initial package checks, identifying reviewers, and moderating the process until the package is accepted by reviewers and transferred to rOpenSci. The number of packages submitted for review has increased every year, resulting in an increased [workload for Editors](/blog/2018/05/03/onboarding-is-work/#work-done-by-editors) and the need to expand the team.
+
+{{< figure src="/img/team/brooke-anderson.jpg" alt="Brooke Anderson" width="250" >}}
+
+**Brooke Anderson** is an Assistant Professor of Epidemiology and a Faculty Associate in the Department of Statistics at [Colorado State University](http://csu-cvmbs.colostate.edu/academics/erhs/Pages/default.aspx), with a PhD in Engineering. She reviewed the [`bomrang`](https://github.com/ropensci/software-review/issues/121), [`charlatan`](https://github.com/ropensci/software-review/issues/94) and [`riem`](https://github.com/ropensci/software-review/issues/39) packages for rOpenSci. Brooke wrote [Mastering Software Development in R](https://bookdown.org/rdpeng/RProgDA/) with Sean Kross and Roger Peng.
+
+> quote from Brooke
+
+Brooke on [GitHub](https://github.com/geanders), [Twitter](https://twitter.com/gbwanderson), [website](http://csu-cvmbs.colostate.edu/academics/erhs/Pages/brooke-anderson.aspx)
+
+{{< figure src="/img/team/melina-vidoni.jpg" alt="Melina Vidoni" width="250" >}}
+
+**Melina Vidoni** has a Postdoctoral Scholarship at [INGAR CONICET-UTN](http://www.ingar.santafe-conicet.gov.ar/), and is an Assistant Lecturer in the Department of Information Systems Engineering at [Universidad Tecnológica Nacional](https://www.frsf.utn.edu.ar/) in Argentina. Statement about R packages? She is a founder of [R-Ladies Santa Fe](https://twitter.com/rladiessantafe) and contributed to the translation into Spanish of [R for Data Science](http://es.r4ds.hadley.nz/).
+
+> quote from Melina
```

Comment:
```suggestion
> I support rOpenSci goals as I strongly believe that software should also be peer-reviewed, so it can help researchers to make the most of their code and data, while also recognising the hard work of the developers behind it. Being an editor here is a great opportunity to learn more about the growing space of research software development in R and its outstanding open-review system, while contributing back to such an incredible active community.
```
add quote from Melina